### PR TITLE
Add jupyterlite_contents config

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 html_logo = "_static/icon.svg"
 
 jupyterlite_config = "jupyterlite_config.json"
+jupyterlite_contents = "./custom_contents"
```
Comment (jasongrout) [inline docs/conf.py:11]: I thought this was a list, not a string?

Comment (martinRenou) [reply docs/conf.py]: Good catch! Thanks

Comment (martinRenou) [reply docs/conf.py]: Yes, that makes sense. I'm pushing a last commit to allow this.

File: src/jupyterlite_sphinx.py [hunk 2]
```diff
@@ -267,7 +270,25 @@ def jupyterlite_build(app: Sphinx, error):
         if app.env.config.jupyterlite_config:
             config = ["--config", app.env.config.jupyterlite_config]
 
+        contents = []
+        if app.env.config.jupyterlite_contents:
+            for content in app.env.config.jupyterlite_contents:
+                contents.extend(["--contents", content])
```
Comment (jasongrout) [inline src/jupyterlite_sphinx.py]:
```suggestion
        contents = []
        dirs = app.env.config.jupyterlite_contents
        if dirs:
            if isinstance(dirs, str):
                dirs = [dirs]
            for content in dirs:
                contents.extend(["--contents", content])

```
This makes it possible for the value to be a single directory, which I bet is a very common case.

File: src/jupyterlite_sphinx.py [hunk 3]
```diff
@@ -267,7 +270,31 @@ def jupyterlite_build(app: Sphinx, error):
...
             config = ["--config", app.env.config.jupyterlite_config]
 
+        contents = []
+        if app.env.config.jupyterlite_contents:
+            jupyterlite_contents = app.env.config.jupyterlite_contents
+
+            if isinstance(contents, str):
+                contents.extend(["--contents", jupyterlite_contents])
+
+            if isinstance(contents, (tuple, list)):
```
Comment (jasongrout) [inline src/jupyterlite_sphinx.py:280]:
```suggestion
            if isinstance(jupyterlite_contents, str):
                contents.extend(["--contents", jupyterlite_contents])

            if isinstance(jupyterlite_contents, (tuple, list)):
```

Comment (martinRenou) [reply src/jupyterlite_sphinx.py:280]: 😅

**Pull request merged**